test/fixtures/dialects/bigquery/create_table_hyphen_project.yml

```diff
@@ -14,7 +14,7 @@ file:
     - table_reference:
       - naked_identifier: project
       - dash: '-'
-      - naked_identifier: name
+      - naked_identifier_part: name
```

Comment:
This change means some rules (e.g. CP02, RF03, RF04...etc.) will no longer match here?

Reply:
ah good point - will revise